Common/src/grid_movement_structure.cpp

```diff
@@ -238,61 +238,63 @@ void CVolumetricMovement::SetVolume_Deformation(CGeometry *geometry, CConfig *co
 
     CSysSolve *system  = new CSysSolve();
     
-    switch (config->GetKind_Deform_Linear_Solver()) {
+    if (LinSysRes.norm() != 0.0){
+      switch (config->GetKind_Deform_Linear_Solver()) {
         
         /*--- Solve the linear system (GMRES with restart) ---*/
         
-      case RESTARTED_FGMRES:
```

Comment:
the whole switch statement is wrapped in an if condition now, thats all (that is needed for multizone deformations). I dont know why github has problems even here ...